src/cli/pack_command.zig

```diff
@@ -2380,6 +2444,25 @@ pub const PackCommand = struct {
 
         Output.flush();
     }
+
+    /// Some files are always packed, even if they are explicitly ignored or not
+    /// included in package.json "files".
+    fn isUnconditionallyIncludedFile(filename: []const u8) bool {
+        // TODO: should this be case insensitive on all platforms?
+        const eql = comptime if (Environment.isLinux)
+            strings.eqlComptime
+        else
+            strings.eqlCaseInsensitiveASCIIICheckLength;
+
+        return filename.len > 5 and (eql(filename, "package.json") or
+            eql(filename, "LICENSE") or
+            eql(filename, "LICENCE") or
+            eql(filename, "README") or
+            filename.len > "README.".len and eql(filename[0.."README.".len], "README."));
+        //or
+        // eql(filename, "CHANGELOG") or
+        // filename.len > "CHANGELOG.".len and eql(filename[0.."CHANGELOG.".len], "CHANGELOG."));
```

Comment:
Looks like originally `CHANGELOG.*` was checked in `isExcluded()`. This looks like it would allow excluding it through .npmignore? We should add a test for this

Reply:
Fixed + test added